Xamarin.Forms.Platform.Android/Cells/EntryCellRenderer.cs

```diff
@@ -40,7 +40,7 @@ public class EntryCellRenderer : CellRenderer
 
 		protected override void OnCellPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
-			if (_view == null || _view.IsDisposed())
+			if (!_view.IsAlive())
```

Comment:
what if `_view == null `  should we try `_view?.IsAlive`?

Reply:
IsAlive is an extension method that returns false if _view is null

Reply:
ah ok!